Bottom: 0128d9cf61845b214043732d3121476128934074
Top:    372f38c2d9b3ecb53493bbef0b37c705f4992ac2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 18:13:14 +0100

Refresh of just-start-chopping-things

---

diff --git a/src/hid/common/extents.c b/src/hid/common/extents.c
index d641f66..6cdc135 100644
--- a/src/hid/common/extents.c
+++ b/src/hid/common/extents.c
@@ -199,7 +199,7 @@ hid_extents_init (void)
 BoxType *
 hid_get_extents (void *item)
 {
-//  BoxType region;
+  BoxType region;
 
   /* As this isn't a real "HID", we need to ensure we are initialised. */
   hid_extents_init ();
@@ -209,13 +209,11 @@ hid_get_extents (void *item)
   box.X2 = -MAXINT;
   box.Y2 = -MAXINT;
 
-#if 0
   region.X1 = -MAXINT;
   region.Y1 = -MAXINT;
   region.X2 = MAXINT;
   region.Y2 = MAXINT;
   hid_expose_callback (&extents_hid, &region, item);
-#endif
 
   return &box;
 }
diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index 4712b1b..44a71fe 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -358,7 +358,6 @@ gcode_finish_png ()
 void
 gcode_start_png_export ()
 {
-#if 0
   BoxType region;
 
   region.X1 = 0;
@@ -371,7 +370,6 @@ gcode_start_png_export ()
   lastcolor = -1;
 
   hid_expose_callback (&gcode_hid, &region, 0);
-#endif
 }
 
 static void
diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 7102f58..baec5ff 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -403,7 +403,7 @@ maybe_close_f (FILE *f)
     }
 }
 
-//static BoxType region;
+static BoxType region;
 
 /* Very similar to layer_type_to_file_name() but appends only a
    three-character suffix compatible with Eagle's defaults.  */
@@ -641,12 +641,10 @@ gerber_do_export (HID_Attr_Val * options)
   lastcap = -1;
   lastcolor = -1;
 
-#if 0
   region.X1 = 0;
   region.Y1 = 0;
   region.X2 = PCB->MaxWidth;
   region.Y2 = PCB->MaxHeight;
-#endif
 
   pagecount = 1;
   resetApertures ();
diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index 7f4ce93..9659b66 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -639,7 +639,6 @@ nelma_finish_png()
 void 
 nelma_start_png_export()
 {
-#if 0
 	BoxType         region;
 
 	region.X1 = 0;
@@ -652,7 +651,6 @@ nelma_start_png_export()
 	lastcolor = -1;
 
 	hid_expose_callback(&nelma_hid, &region, 0);
-#endif
 }
 
 static void
